test/wasm/rot.wast

```diff
@@ -0,0 +1,15 @@
+;;-------------------------------------------------------------------------------------------------------
+;; Copyright (C) Microsoft and contributors. All rights reserved.
```

Comment:
At first I couldn't figure out what the problem here was.

You'll need to change this line to

```
;; Copyright (C) Microsoft Corporation and contributors. All rights reserved.
```